tox.ini

```
@@ -89,6 +89,9 @@ commands =
     snowparkpandasnotdoctest: {env:MODIN_PYTEST_CMD} --durations=20 -m "{env:SNOWFLAKE_TEST_TYPE}" {posargs:} {env:SNOW_1314507_WORKAROUND_RERUN_FLAGS} tests/unit/modin tests/integ/modin
     # This one only run doctest but we still need to include the tests folder to let tests/conftest.py to mark the doctest files for us
     snowparkpandasdoctest: {env:MODIN_PYTEST_CMD} --durations=20 -m "{env:SNOWFLAKE_TEST_TYPE}" {posargs:} src/snowflake/snowpark/modin/pandas/ tests/unit/modin
+    # This one is only called by jenkins job and the only difference from `snowparkpandasnotdoctest` is that it uses
+    # SNOWFLAKE_PYTEST_NO_COV_CMD instead of SNOWFLAKE_PYTEST_CMD
+    snowparkpandasjenkins: {env:SNOWFLAKE_PYTEST_NO_COV_CMD} --durations=20 -m "{env:SNOWFLAKE_TEST_TYPE}" {posargs:} {env:SNOW_1314507_WORKAROUND_RERUN_FLAGS} tests/unit/modin tests/integ/modin
```

Under Line 63, I would do:
`MODIN_PYTEST_NO_COV_CMD = pytest {env:SNOWFLAKE_PYTEST_VERBOSITY:} {env:SNOWFLAKE_PYTEST_PARALLELISM:} --ignore=tests/resources`

Then do:
```suggestion
    snowparkpandasjenkins: {env:MODIN_PYTEST_NO_COV_CMD} --durations=20 -m "{env:SNOWFLAKE_TEST_TYPE}" {posargs:} {env:SNOW_1314507_WORKAROUND_RERUN_FLAGS} tests/unit/modin tests/integ/modin
```

You also need to update this test to include MODIN_PYTEST_NO_COV_CMD. See this: https://github.com/snowflakedb/snowpark-python/pull/1421/files#diff-644ecac66746749550c23f6a19cb2ea505c2208548e4606d8ee9209c0c174baeR887

---

done by merging the other branch.